Streetcode/Streetcode.BLL/Mapping/Streetcode/RelatedFigureProfile.cs

```diff
@@ -1,13 +1,20 @@
 ﻿using AutoMapper;
 using Streetcode.BLL.DTO.Streetcode;
-using Streetcode.DAL.Entities.Streetcode;
+using Streetcode.DAL.Entities.Streetcode.Types;
 
 namespace Streetcode.BLL.Mapping.Streetcode;
 
 public class RelatedFigureProfile : Profile
 {
     public RelatedFigureProfile()
     {
-        CreateMap<RelatedFigure, RelatedFigureDTO>().ReverseMap();
+        CreateMap<EventStreetcode, RelatedFigureDTO>()
+            .ForPath(dto => dto.Title, conf => conf.MapFrom(e => e.Title))
+            .ForPath(dto => dto.ImageId, conf => conf.MapFrom(e => e.Images.Select(i => i.Id).FirstOrDefault()));
+
+        CreateMap<PersonStreetcode, RelatedFigureDTO>()
+            .ForPath(dto => dto.Title, conf => conf
+                .MapFrom(e => (e.Rank == null) ? e.FirstName + " " + e.LastName : e.Rank + " " + e.FirstName + " " + e.LastName))
```

Comment:
Consider using string interpolation here to achieve much cleaner and concise statement.